src/Tasks/Copy.cs

```diff
@@ -559,77 +607,23 @@ private bool CopyParallel(
 
             // Lockless flags updated from each thread - each needs to be a processor word for atomicity.
             var successFlags = new IntPtr[DestinationFiles.Length];
-            var actionBlockOptions = new ExecutionDataflowBlockOptions
-            {
-                MaxDegreeOfParallelism = parallelism,
```

Comment:
the PR lost the logic for `parallelism` variable values, this should be cleaned up to be transparent that it does nothing apart from deciding if execution is synchronous or parallel (with the default values).

Reply:
Should be updated now.